### PR TITLE
is palindrome zadatak

### PR DESCRIPTION
**Pull request by petrovicdarko1234**
https://leetcode.com/problems/palindrome-number/description/

### CODE REVIEW

File: problems/28.Palindrome Number.gs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    while (i != j && j > i) {
+        if (niz[i] != niz[j]) {
+            return false
+        }
+        i++
+        j--
+    }
+    return true
+};
+function uNiz(x: number): number[] {
```
Comment (sloba-byte) [inline problems/28.Palindrome Number.gs]: how many times do I need to say write in English? It's not a problem I will do it each time :)

File: problems/28.Palindrome Number.gs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    let str: string = x.toString()
+    let niz: number[] = new Array(str.length)
```
Comment (sloba-byte) [inline problems/28.Palindrome Number.gs]: english never? death to english

File: problems/28.Palindrome Number.gs [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        if (niz[i] != niz[j]) {
+            return false
+        }
+        i++
+        j--
+    }
+    return true
+};
+function uNiz(x: number): number[] {
+    let str: string = x.toString()
```
Comment (sloba-byte) [inline problems/28.Palindrome Number.gs]: This is extremely inefficient way to extract digits from string...
Try be doing this:
x%10 => will give you last digit
x = x / 10 => will remove last digit
Do the above two operation as long as X  != 0; Much faster then converting to string...

Comment (sloba-byte) [reply problems/28.Palindrome Number.gs]: and less memory a lot less memory

File: problems/28.Palindrome Number.gs [hunk 4]
```diff
@@ -0,0 +1,31 @@
+function isPalindrome(x: number): boolean {
+    if (x < 0) {
+        return false
+    }
+
+    let niz = uNiz(x)
+    let i = 0
+    let j = niz.length - 1
+
+    while (i != j && j > i) {
```
Comment (sloba-byte) [inline problems/28.Palindrome Number.gs:11]: Try writing this with For, just to see if you can do it same logic with two counters.

**Review by sloba-byte — CHANGES_REQUESTED** [review]

File: problems/28.Palindrome Number.gs (hunk 4 above)
Comment (petrovicdarko1234) [reply problems/28.Palindrome Number.gs:11]: ne biva

**Review by sloba-byte — APPROVED** [review]
good job

**Pull request merged**